Makefile

```diff
@@ -38,8 +39,8 @@ test: structure-test docker-tests
 # push docker images to mlregistry.marklogic.com
 #***************************************************************************
 push-mlregistry:
-	docker tag ${REPONAME}/marklogic-server-centos:${version} ml-docker-dev.marklogic.com/${repoDir}/marklogic-server-centos:${version}
-	docker push ml-docker-dev.marklogic.com/${repoDir}/marklogic-server-centos:${version}
+	docker tag ${REPONAME}/marklogic-server-centos:${version} ${docker_regestry}/${repoDir}/marklogic-server-centos:${version}
+	docker push ${docker_regestry}/${repoDir}/marklogic-server-centos:${version}
```

Comment:
Typo here as well:
regestry = registry ?

Reply:
Yes, and in three other places.